.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: node_js
 
+git:
+  depth: 3
```

Comment:
Depth 3: [1.55s](https://travis-ci.org/zurb/foundation-sites/jobs/372788385#L416)
Depth 50: [3.15s](https://travis-ci.org/zurb/foundation-sites/jobs/372717914#L416)

> Please note that if you use a depth of 1 and have a queue of jobs, Travis CI won’t build commits that are in the queue when you push a new commit.
> -- https://docs.travis-ci.com/user/customizing-the-build#Git-Clone-Depth

So this may introduce some dysfunctioning with Travis, and give almost no benetifs.

Reply:
> Please note that if you use a depth of 1 and have a queue of jobs

We use depth: 3, not 1. 1 would be a shallow clone.

Reply:
Hm, what changes did you request or did you just ant to comment? I see no proposed changes.

Reply:
https://www.perforce.com/blog/git-beyond-basics-using-shallow-clones

Reply:
> We use depth: 3, not 1. 1 would be problematic.

This is still problematic with 3+ concurrents Travis jobs. It's very unlikely but I would not risk it to save 2s of test time.

Reply:
> We simply don't need it.

Ok.